CrazyCanvas/Source/States/ServerState.cpp

```diff
@@ -65,6 +66,8 @@ bool ServerState::OnServerDiscoveryPreTransmit(const LambdaEngine::ServerDiscove
 	ServerBase* pServer = event.pServer;
 
 	pEncoder->WriteUInt8(pServer->GetClientCount());
+	pEncoder->WriteString(m_ServerName);
+	pEncoder->WriteString("Bajskorv");
```

Comment:
Maybe choose a better Test string 
```suggestion
	pEncoder->WriteString("This Is the Map Name");
```

Reply:
You are correct, but you can't say that it didn't make you smile a little.